internal/proxy/proxy.go

```diff
@@ -34,6 +38,30 @@ func New(
 	}
 }
 
+type StatusResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  struct {
+		NodeInfo struct {
```

Comment:
move declaration of nested structs into dedicated types

Reply:
Thank you for your review, I'll change this asap